src/textarea/Textarea.story.mdx

```diff
@@ -88,5 +92,19 @@ stories.add('Text area with icon and fit to content', () => (
         window.alert('button clicked')
       }}
     />
-  </Section>
-))
+  </Story>
+</Preview>
+
+## Specifications
+
+This component is a text area and accept icons.
+
+## Usage
+
+```jsx
+import { Textarea } from '@blablacar/ui-library/build/textarea'
+
+<Textarea />
```

Comment:
it's cool to have more attributes here to be able to copy/paste the whole version